data/mods/blazemod/blaze_ammo_recipes.json

```diff
@@ -135,7 +135,7 @@
   "reversible": false,
   "autolearn": true,
   "qualities" : [
-    {"id":"CUT","level":1,"amount":1}
+    {"id":"CUT","level":1}
```

Comment:
JSON style would be all on one line where there is just one quality (like the entry above this). Correct indentation and formatting greatly helps readability